python/miniapp/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(PYMINIAPP_SOURCES
```

Comment:
I would expect the same pattern to develop as the c++ side: multiple miniapps. So the directory structure could mimic this?

Reply:
Yes, this is what I've done on the feature/python branch which add pyminiapp + pyarbor